src/org/abymaly/code/Sieve.java

```diff
@@ -0,0 +1,58 @@
+package org.abymaly.code;
+
+public class Sieve {
+
+    public int[] createSieve(int maxVal) {
+
+        // TODO: takie podstawienie size jest raczej bez sensu, ale to jest jakiś początek ograniczenia wielkości tablicy do pierwiastka szukanej wartości.
```

Comment:
To jest tak, że albo szybko albo wydajnie pamięciowo.
Sito jest pamięciożerne, ale sprawdzenie, czy jest pierwsza ma złożoność O(1)